### PR TITLE
Update stat_tracker spec, change all stat class to modules/mod spec

### PR DESCRIPTION
**Pull request by Henchworm**
FULL COVERAGE

### CODE REVIEW

File: lib/game_stats.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-class GameStats
-  
+module GameStats
```
Comment (jaw772) [inline lib/game_stats.rb:1]: nice revision. good idea!

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -7,13 +7,21 @@
 RSpec.describe Class do
 
   it "attributes" do
-    stattracker= StatTracker.new
-    expect(stattracker.games).to eq(nil)
-    expect(stattracker.teams).to eq(nil)
-    expect(stattracker.game_teams).to eq(nil)
+    game_path = './data/games.csv'
```
Comment (tjroeder) [inline spec/stat_tracker_spec.rb:10]: Good changes to the testing.

Comment (jaw772) [inline spec/stat_tracker_spec.rb:10]: nice transition and clean up

Comment (jacksonvaldez) [inline spec/stat_tracker_spec.rb:10]: I like how you removed the initialize test and included it in the from_csv test.

**Review by jaw772 — APPROVED** [review]

**Review by tjroeder — APPROVED** [review]
Good job updating the classes and tests to modules rather than classes.

File: lib/league_stats.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
-class LeagueStats
-  
+module LeagueStats
```
Comment (jacksonvaldez) [inline lib/league_stats.rb:1]: It makes sense that the Stats files are modules. It is a good way of leveraging benefits of OOP and not putting everything in StatTracker

**Review by jacksonvaldez — APPROVED** [review]

**Pull request merged**